docker-compose-elastic-cloud.yml

```diff
@@ -0,0 +1,110 @@
+version: "2.1"
```

Comment:
this file is not related to structured log, is a docker-compose file to use Elastic Cloud as stack

Reply:
For the record, 2.x is the version that enables the `depends_on:` directive with the `condition`